Scenarios/GhTokenWorkflow.md

```diff
@@ -0,0 +1,90 @@
+# Creating a GhTokenWorkflow secret
+
+Many AL-Go for GitHub workflows will require a GhTokenWorkflow secret. The GhTokenWorkflow secret should contain either a Personal Access Token with a specific set of permissions or it should be a compressed json secret containing the GitHub App Client Id and a Private Key of an installed GitHub app, with the same permissions.
+
+The Personal Access Token comes in two flavors as well: Classic or Fine-grained.
+
+> [!NOTE]
+> The permissions assigned to the GitHub App or PAT specified in the GhTokenWorkflow secret should not be confused with the repositories for which the GhTokenWorkflow secret is made available.
+> If the secret is available to a repository, for which the GitHub App or PAT haven't been given permissions, it will not work.
+
+## GitHub App Authentication
```

Comment:
You can add "(recommended)" or something to denote that this is the recommended way of doing GitHub auth.